app/services/authorizer.rb

```diff
@@ -104,8 +104,13 @@ def build_filtered_scope_components(resource_class, all_filters, options)
   def build_scoped_search_condition(filters)
     raise ArgumentError if filters.blank?
 
-    strings = filters.map { |f| "(#{f.search_condition.presence || '1=1'})" }
-    strings.join(' OR ')
+    conditions = filters.map do |f|
+      return '' if f.search_condition.blank?
```

Comment:
return will return from the method, that's not what we want I'd guess :thinking: I'd rather go with something like
```ruby
conditions = filters.map(&:search_condition).reject(&:blank?)
# OR
conditions = filters.map { |f| f.search_condition.presence }.compact
conditions.map { |condition| "(#{condition})" }.join(' OR ')
```